[2025-09-24T09:09:34.930+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_h1 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:09:34.951+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_h1 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:09:34.952+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T09:09:34.975+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cleanse_data_h1> on 2025-09-24 08:00:00+00:00
[2025-09-24T09:09:34.986+0000] {standard_task_runner.py:60} INFO - Started process 1659 to run task
[2025-09-24T09:09:34.991+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'cleanse_data_h1', 'scheduled__2025-09-24T08:00:00+00:00', '--job-id', '248', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmpc8yqp6dn']
[2025-09-24T09:09:34.993+0000] {standard_task_runner.py:88} INFO - Job 248: Subtask cleanse_data_h1
[2025-09-24T09:09:35.072+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_h1 scheduled__2025-09-24T08:00:00+00:00 [running]> on host feaba2204362
[2025-09-24T09:09:35.221+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='cleanse_data_h1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-24T08:00:00+00:00'
[2025-09-24T09:09:35.429+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:09:35.450+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=cleanse_data_h1, execution_date=20250924T080000, start_date=20250924T090934, end_date=20250924T090935
[2025-09-24T09:09:35.615+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:09:35.702+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-24T09:59:31.083+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_h1 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:59:31.101+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_h1 scheduled__2025-09-24T08:00:00+00:00 [queued]>
[2025-09-24T09:59:31.104+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-24T09:59:31.127+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): cleanse_data_h1> on 2025-09-24 08:00:00+00:00
[2025-09-24T09:59:31.150+0000] {standard_task_runner.py:60} INFO - Started process 966 to run task
[2025-09-24T09:59:31.155+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'cleanse_data_h1', 'scheduled__2025-09-24T08:00:00+00:00', '--job-id', '286', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmp5vq3axyx']
[2025-09-24T09:59:31.157+0000] {standard_task_runner.py:88} INFO - Job 286: Subtask cleanse_data_h1
[2025-09-24T09:59:31.250+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.cleanse_data_h1 scheduled__2025-09-24T08:00:00+00:00 [running]> on host 5d539c157d54
[2025-09-24T09:59:31.341+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='cleanse_data_h1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-24T08:00:00+00:00'
[2025-09-24T09:59:31.483+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:59:31.490+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=cleanse_data_h1, execution_date=20250924T080000, start_date=20250924T095931, end_date=20250924T095931
[2025-09-24T09:59:31.540+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:59:31.558+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
